go.mod

```diff
@@ -345,7 +346,7 @@ require (
 	github.com/shurcooL/httpfs v0.0.0-20230704072500-f1e31cf0ba5c // indirect
 	github.com/shurcooL/httpgzip v0.0.0-20190720172056-320755c1c1b0 // indirect
 	github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546 // indirect
-	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726 // indirect
+	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726
```

Comment:
please move it to direct group to keep it pretty.

Reply:
Removed https://github.com/pingcap/tiflow/pull/11671/commits/8e601b3d1887ad9b03a38064b5e1669a4a1b81c5